front/src/iframe_api.ts

```diff
@@ -13,6 +13,10 @@ interface WorkAdventureApi {
     onEnterZone(name: string, callback: () => void): void;
     onLeaveZone(name: string, callback: () => void): void;
     openPopup(targetObject: string, message: string, buttons: ButtonDescriptor[]): Popup;
+    disablePlayerControl() : void;
+    enablePlayerControl() : void;
```

Comment:
I would probably use `restorePlayerControls` (because it is by default enabled) and also, it matches the name of the methods in UserInputManager